src/classes/SRVC_Contact_PrimaryAffiliations.cls

```diff
@@ -170,27 +174,25 @@ public virtual with sharing class SRVC_Contact_PrimaryAffiliations {
 
         try{
             for (MODL_PrimaryAfflMapper eachModel: afflMappingsModelsList) {
-
                 if (newContact.get(eachModel.getContactFieldApiName()) == oldContact.get(eachModel.getContactFieldApiName())) {
                     continue;
                 }
-
+               
                 if (newContact.get(eachModel.getContactFieldApiName()) != null) {
-                    if (eachModel.getIsValid() == false){
-                        newContact.addError(Label.afflAccoutMappingError);
-                        throw new ERR_ExceptionHandler.AffAccountRecordTypeMappingsException(Label.afflAccoutMappingError);
+                    if (eachModel.getIsValid() == false || String.isBlank(eachModel.getInvalidErrorMessage()) == false){ 
```

Comment:
Want to make sure: based on eda style guide, should we use `myString.isBlank()` instead?

Reply:
Also, we are throwing exceptions when model is valid but there is invalid error message? That doesn't sound so right?

Reply:
I think the syntax for the isBlank() method of the String class is String.isBlank(myString).  I tried myString.isBlank() and I get an error.

Reply:
yeah we will need to update the style guide then. Also, really up to you, but isNotBlank == true would be more readable?

Reply:
isBlank(thing) == false may be clear enough. Do we want to add a local variable to accomplish this? Is it adding enough value to be incorporated and named?

Reply:
> Also, we are throwing exceptions when model is valid but there is invalid error message? That doesn't sound so right?
What to bring this up again. Do we want to allow model with isValid =true but invalidErrorMsg not blank to throw exceptions?
@m-kinkade @MatthewBlanski

Reply:
In this scenario, the isValid flag is used to indicate whether the Record Type mapping in affiliation mappings is a valid Account Record Type when validation is enabled.  The error message variable is used to capture any errors associated with the model (in this case, invalid Record Type or Contact field mapping).  Maybe we could rename isValid to be more specific, like isRecordTypeValid instead?

Reply:
@MatthewBlanski curious what's your opinion on this.

Reply:
For the purposes of this PR, let's leave off of that variable. Eventually we will want to modify that value to capture a variety of states. We may wish to do this within a context of when we address a mapper when we are working off of the Affiliation object.

If you think the LOE is low, then please, go ahead and rename it.